diff --git a/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js b/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
index 2fe6a6e..7c8e7de 100644
--- a/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
+++ b/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
@@ -33,6 +33,7 @@ class Prayer extends _postgres.PostgresDataSource {
       Notification.createAndSend({
         title: notificationText,
         personId,
+        body: 'Pray for others?',
         type: 'PRAYER'
       });
     });
@@ -64,8 +65,9 @@ class Prayer extends _postgres.PostgresDataSource {
     });
     followers.forEach(follow => {
       Notification.createAndSend({
-        title: `${firstName} ${lastName} is asking for prayer. Will you pray for them?`,
+        title: `${firstName} ${lastName} is asking for prayer.`,
         personId: follow.requestPersonId,
+        body: 'Will you pray for them?',
         type: 'PRAYER'
       });
     });
@@ -138,10 +140,8 @@ class Prayer extends _postgres.PostgresDataSource {
     const prayerRequest = await this.getFromId(prayerId);
     await prayerRequest.addPrayedUser(currentPerson);
     const usersPrayed = await prayerRequest.getPrayedUsers();
-
-    if (usersPrayed.length === 1) {
+      console.log("NUMBER OF USERS PRAYED", usersPrayed.length);
       this.sendPrayingNotification(prayerRequest.personId);
-    }
   }
 
 }
