diff --git a/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js b/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
index 2fe6a6e..a0e7c8c 100644
--- a/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
+++ b/node_modules/@apollosproject/data-connector-postgres/lib/prayers/dataSource.js
@@ -64,8 +64,9 @@ class Prayer extends _postgres.PostgresDataSource {
     });
     followers.forEach(follow => {
       Notification.createAndSend({
-        title: `${firstName} ${lastName} is asking for prayer. Will you pray for them?`,
+        title: `${firstName} ${lastName} is asking for prayer.`,
         personId: follow.requestPersonId,
+        body: 'Will you pray for them?',
         type: 'PRAYER'
       });
     });
@@ -138,7 +139,7 @@ class Prayer extends _postgres.PostgresDataSource {
     const prayerRequest = await this.getFromId(prayerId);
     await prayerRequest.addPrayedUser(currentPerson);
     const usersPrayed = await prayerRequest.getPrayedUsers();
-
+      console.log("NUMBER OF USERS PRAYED", usersPrayed.length);
     if (usersPrayed.length === 1) {
       this.sendPrayingNotification(prayerRequest.personId);
     }
